src/test/resources/decodes/db/CsvPipeDelimiterDocSample.input

```diff
@@ -0,0 +1,4 @@
+ignored header line
+03/01/2014 12:00:00 23.95| 17.2| 8252| 0| 0| 98.1
```

Comment:
why not a pipe delimiter after the time-stamp; before the value?

Reply:
it appears I've simply forgotten it... though now I need to check the csv I copied that from.

Reply:
Okay, just checked the decodes script processed the date first and then passes the remaining bits to the CSV, so the CSV function only starts after the date.